chain/network/Cargo.toml

```diff
@@ -11,11 +11,11 @@ borsh = { version = "0.9", features = ["rc"]}
 byteorder = "1.2"
 bytes = "1"
 bytesize = "1.1"
-cached = "0.23"
 chrono = { version = "0.4.4", features = ["serde"] }
 conqueue = "0.4.0"
 futures = "0.3"
 lazy_static = "1.4"
+lru = "=0.6.5"
```

Comment:
No need for `=` here, pinning is done by Cargo.lock. To ask cargo to put specific verison in cargo lock, use the following command: 

```console
$ cargo update -p lru --precise 0.6.6
```

Reply:
I think it's better to specify `0.6.5`, because the `0.6.6` is broken.

Reply:
I changed it to `lru = "0.6.5"`

Reply:
I mean, we should say `lru = "0.6.5"` in Cargo.toml, and let Cargo.lock to take care about `=`.